server/src/database/gameDB.ts

```diff
@@ -22,21 +22,31 @@ const gameComputer: Computer<BaseUnifiedGame, UnifiedGameComputed> = {
 type WatchableGame = WatchableResource<BaseUnifiedGame, UnifiedGameComputed>
 
 const gameDB: Record<string, WatchableGame> = {}
+const storage = new StoreFile<BaseUnifiedGame>('game', new RemoteStorage())
 
-export function gameExists (gameId: string): boolean {
-  return !!gameDB[gameId]
+export async function gameExists (gameId: string): Promise<boolean> {
+  if (gameDB[gameId]) return true
+
+  const gameFromStorage = await storage.getFile(gameId)
+  if (gameFromStorage) {
+    gameDB[gameId] = new WatchableResource(gameFromStorage, gameComputer)
+    gameDB[gameId].subscribe((value) => { void storage.putFile(gameId, value as BaseUnifiedGame) })
```

Comment:
I suspect this will save the whole `UnifiedGame` to storage, but otherwise shouldn't create an issue